master/internal/resourcemanagers/priority.go

```diff
@@ -383,3 +383,39 @@ func taskFilter(label string, zeroSlots bool) func(*sproto.AllocateRequest) bool
 		return request.Label == label && (request.SlotsNeeded == 0) == zeroSlots
 	}
 }
+
+func sortTasks(
```

Comment:
nit: will this be used in the agent-based priority scheduler?

Reply:
not at the moment - the agent one sets the job status, whereas in k8s we must observe the pod status